Makefile

```diff
@@ -39,9 +40,11 @@ test:
 .PHONY: clean
 clean:
 	- rm -f tc-redirect-tap
+	- rm -rf $(GOBIN)
 
 deps:
 	echo
 
 lint:
-	echo
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(GOBIN) latest
```

Comment:
Here instead of latest let's pin the version to a specific version for build reproducibility. `v1.46.2` should be fine.